app/assets/javascripts/models/story.js

```diff
@@ -153,19 +152,28 @@ var Story = module.exports = Backbone.Model.extend({
 
   // List available state transitions for this story
   events: function() {
-    switch (this.get('state')) {
-      case 'started':
-        return ["finish"];
-      case 'finished':
-        return ["deliver"];
-      case 'delivered':
-        return ["accept", "reject"];
-      case 'rejected':
-        return ["restart"];
-      case 'accepted':
-        return [];
-      default:
-        return ["start"];
+    var storyType = this.get('story_type');
+
+    if (storyType != 'related')
```

Comment:
You don't need the else here, you can rewrite this logic to something like this:

```
if (storyType != 'related') {
  return [];
}

switch...
```